src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -244,6 +310,25 @@ const SampleChatInfoScreen = () => {
           </View>
         </>
       )}
+      <ModalAction
+        onCloseModal={handleCloseSelectUser}
+        selectedUser={selectedUser}
+        isOpen={openModal}
+        onPress={handlePressPopup}
+        name={selectedUser?.user?.username || selectedUser?.user?.name}
+      />
+      <ModalActionAnonymous
+        name={
+          selectedUser?.user?.name !== ANONYMOUS_USER
+            ? selectedUser?.user?.name
+            : selectedUser?.user?.username
+        }
+        isOpen={isAnonymousModalOpen}
+        onCloseModal={handleCloseSelectUser}
+        selectedUser={selectedUser}
+        onPress={handlePressPopup}
+      />
+      <BlockComponent ref={blockModalRef} screen="group_info" />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Modal components have been added to the return statement of `SampleChatInfoScreen`. Ensure that these modals work as expected and handle all edge cases.


<!-- This is an auto-generated comment by OSS CodeRabbit -->